doc/quickstart.md

```diff
@@ -3,38 +3,44 @@
 These instructions will walk you through setting up Lean using the "basic" setup and VS Code as the editor.
 See [Setup](./setup.md) for other ways and more details on setting up Lean.
 
-1. Install the latest Lean 4 nightly through [`elan`](https://github.com/leanprover/elan): in any bash-compatible shell, run
-    ```sh
-    curl https://raw.githubusercontent.com/leanprover/elan/master/elan-init.sh -sSf | sh -s -- --default-toolchain leanprover/lean4:nightly
-    ```
-    On Windows, instead run in `cmd`
-    ```sh
-    curl -O --location https://raw.githubusercontent.com/leanprover/elan/master/elan-init.ps1
-    powershell -f elan-init.ps1 --default-toolchain leanprover/lean4:nightly
-    del elan-init.ps1
-    ```
-    See the [elan repo](https://github.com/leanprover/elan) for other installation options and details.
-
 1. Install [VS Code](https://code.visualstudio.com/).
 
-1. Create a new folder called `Foo` and type:
+1. Create a new folder called `foo` and add a file named `hello.lean`
+containing the following:
+
     ```lean
-    lake init Foo
+    import Lake
+    #eval Lake.leanVersionString
+    ```
+
+    and a file named `lean-toolchain` containing just this:
```

Comment:
Hmm, I concede that this is hardly better than before. What is still missing in the extension to make this work purely in VS Code, ideally without saving any file at all?

Reply:
I think I must have missed the comment you're replying to, what was the previous proposal?  And what do you have in mind for the automatic set up?

Reply:
The previous version installed elan from the cmdline to then run `lake init`. The ideal, no-hassle on-boarding I have in mind is:
1. install extension
2. File > New File
3. either "Select a language" > lean4 or save as `.lean` (does the latter default to Lean 4 when the Lean 3 extension is not installed?)
4. it should just work, where "just work" probably implies defaulting to nightly

Reply:
Ok, love to get your input on this on then https://github.com/leanprover/vscode-lean4/pull/83

Reply:
Is this resolved now on the extension side?

Reply:
Right, I will update this doc and PR for the new behavior of the vscode extension...

Reply:
Ok, PR is updated.  I had to switch to `Leanpkg.leanVersionString` because this seems `Lake.leanVersionString` has been removed.  
```
unknown identifier 'Lake.leanVersionString'
```

Reply:
@lovettchris why  not just use `Lean.versionString`?

Reply:
Doh!  Done, had to update the screenshot too.

Reply:
See [quickstart.md](https://github.com/leanprover/lean4/blob/2286262eaa08e77ed0b55ade3d2a95c8801342d3/doc/quickstart.md)

Reply:
Why do you insist on using a named file? I thought unnamed files just worked now?

Reply:
Are you talking about moving step (2) after the current step (3) and making it like this?

3. Start VS code and select File/New File
4. Click "Select a Language" and type in "lean4" then paste in the following program:
```lean
import Lean
#eval Lean.versionString
```

Problem is I just tried this and it doesn't work:
```
'lean' is not recognized as an internal or external command,
operable program or batch file.
[Info  - 11:54:27 AM] Connection to server got closed. Server will restart.
'lean' is not recognized as an internal or external command,
```
we'll need a new issue opened to fix this.  But quite frankly I think creating a real file on disk is going to prepare them a lot better for playing with Lake Projects.

Reply:
So https://github.com/leanprover/vscode-lean4/issues/92 isn't fixed after all?

Reply:
That scenario works, just not together with bootstrapping apparently.

Reply:
my PR on "[multi-folder support](https://github.com/leanprover/vscode-lean4/pull/104)" fixes it though.

Reply:
Nice. I really think the steps in this file should be the quickest possible to get to a working Lean version, *especially* as long as we don't have a working web version. I don't think most people turn their very first Lean file into a package later, but we can add steps for that as well.

Reply:
I agree, see updated [quickstart.md](https://github.com/leanprover/lean4/blob/2286262eaa08e77ed0b55ade3d2a95c8801342d3/doc/quickstart.md)